book/website/reproducible-research/reviewing/reviewing-recommend.md

```diff
@@ -5,26 +5,39 @@
-the new changes in the code to do a good job in a reasonable amount of time.
 
 In contrast, large-scale development projects will likely have existing, concrete rules for
 how reviewers are allocated to individual pull requests.
 These rules serve to balance the group workload and to maximise the various benefits of the process to the project and its participants.
 The very largest projects may even have dedicated staff - or teams of staff - to act as reviewers.
 Typically, code reviews can only be performed by an authorised subset of contributors within larger projects.
 
+If you are a contributor to a more loosely managed or distributed project, a good rule of thumb is to aim to perform reviews in similar quantities to the material you generate for review.
+
 For projects where multiple rounds of review on similar material are likely and long development cycles are anticipated, a degree of strategic thinking on who completes reviews is sensible.
 A single reviewer is likely to be able to make comments on code they have reviewed before much more efficiently.
 However, letting reviewer-coder pairs like this persist is generally a bad idea, as it can lead to the same kinds of groupthink that the review process is designed to avoid in the first place.
 
+Open source projects often also have their own dynamics.
+Contributions may be split between a set of core maintainers, and a larger set of more dispersed contributors, some of whom may have very limited interactions with a project.
+In open source projects it is typically the case that anyone can review a pull request, just as anyone can create one.
+Those who have merge privileges, often limited to core maintainers, inevitably have the final say over what gets added to the code, and so also have the largest role when it comes to code review.
+
 (rr-reviewing-recommendation-be-nice)=
 ## Be Nice!
 
-As with all open-source and collaborative enterprises, good internet etiquette makes the whole process go more smoothly.
+As with all open source and collaborative enterprises, good internet etiquette makes the whole process go more smoothly.
 Perhaps most importantly, always assume good faith on both sides of the review interaction, and always be constructive.
 These principles are true for the review process beyond almost any other project aspect, since it necessarily involves criticism, potentially between two complete strangers.
 
+Although the review process is asymmetrical, the challenges go in both directions: reviewees may feel their ideas are being misunderstood or criticised unjustly; reviewer's may feel their input is being ignored.
+It's important that all parties in the review process avoid being overly pendantic or territorial.
+
+For an open source project hoping to grow a community it's also important to understand that it can be better to accept poor code contributions than to alienate potential collaborators.
```

Comment:
We discussed that we would prefer a different wording in this sentence. Accepting “poor code” and “alienating contributors” are rather extreme. It is good to mention the tradeoff between code quality and “being nice” to grow the community. Suggestion:

"External contributions might not be at at the standards you set for your own code. When you reject a pull request, make sure to explain, in a constructive manner, why you chose to reject it and what could be done to improve it. If you want to grow your community, this could be a good opportunity to mentor a new contributor."